src/link/Coff/lld.zig

```diff
@@ -226,6 +227,11 @@ pub fn linkWithLLD(self: *Coff, comp: *Compilation, prog_node: *std.Progress.Nod
         if (!self.base.options.dynamicbase) {
             try argv.append("-dynamicbase:NO");
         }
+        if (self.base.options.allow_shlib_undefined) |allow_shlib_undefined| {
+            if (allow_shlib_undefined) {
```

Comment:
I'm going to merge this as-is since it's perfectly fine, but note that this is possible in zig:

```suggestion
        if (self.base.options.allow_shlib_undefined == true) {
```